src/components/Navigator/NavigatorCard.vue

```diff
@@ -933,6 +933,7 @@ export default {
       this.$refs.scroller.$el.scrollBy({
         top: offsetHeight * positionIndex,
         left: 0,
+        behavior: 'smooth',
```

Comment:
I think this should not have smooth scrolling. Its causing weird jumps as we are sampling the item if its in the viewport or not, too fast.